### DISCUSSION TITLE
Custom title screen song for OpenRCT2

### DISCUSSION THREAD

**Discussion by Nubbie** [OpenRCT2/OpenRCT2#15719] [Feature requests] [answered]
As we have a custom title screen - Why not a custom song

The song should be a alteration on the RCT2 song
The only one I had in mind would be https://youtu.be/tvD2cXa4KJM which is a remix off the RCT2 song

**Comment by ccoors** [comment]
I don't think that's possible without getting permission from Atari/Chris Sawyer/Allister Brimble, whoever holds the rights.

**Comment by IntelOrca** [comment]
You can replace CSS17 yourself if you want to do this.

**Comment by Nubbie** [comment]
@IntelOrca Indeed, which I did, but thought it as a default thing

**Comment by ccoors** [comment]
Make an issue here:
https://github.com/ccoors/OpenRCT2-OpenMusic
Maybe, in some distant future, someone or even I can make a custom theme song.

**Comment by Ryder17z** [comment]
Would be nice to have in the future

**Comment by J-PIE-314** [comment, marked as answer]
This feature was implemented in OpenRCT2 via https://github.com/OpenRCT2/OpenMusic/pull/41.
See also: https://www.youtube.com/watch?v=NkvaRXyJOTM
